ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -1237,6 +1285,71 @@ impl LookupGlobalInst {
         self.global_decl_idx
     }
 }
+///
+/// The operands for a [Inst::Call]
```

Comment:
Update the docstring to reference a `IndirectCall` instruction?

Reply:
[31e98ca](https://github.com/ykjit/yk/pull/1141/commits/31e98cac0bc2a5989282aa4fc0c0c2d0537230a5)